### DISCUSSION TITLE
rolling supported python version(>=3.10) ?

### DISCUSSION THREAD

**Discussion by Qervas** [inviwo/inviwo#1582] [General]
Hey guys. I was trying to build the project under Python 3.12 then I found out the Pybind modules only support 3.10. Luckily, virtual env solved this issue. May I ask if this project will be considered for a newer version of Python integrated into the system? Thx

**Comment by petersteneteg** [comment]
Hi,

Did you build using the old submodules or vcpkg? with vcpkg python 3.12 should work. And the latest master should also handle numpy 2.0. But the old submodules have not been update for some time. And will also be removed soon.
